laravel-app/app/Http/Controllers/ProfileController.php

```diff
@@ -74,6 +80,7 @@ public function show(int $userId): RedirectResponse|View
         return view('profile.show', [
             'user' => $user,
             'likesCount' => $likesCount,
```

Comment:
just wondering why do you still need this likesCount if you already are adding it to the posts using with.

Reply:
i added likesCount here sir so I will not have to sum the likes at the show blade

Reply:
I see. okay so the likecounts the total overall of the likes that the user received. the the post is the just for the specific post

Reply:
yes sir 🙇

Reply:
I see okay then maybe that function of getting the sum of a user you can put it in the model of the user??

Reply:
always remember the concept of fat model , thin controller

Reply:
definitely sir, added accessor to the user model thank you ❤️